src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -488,6 +488,18 @@ const vector<Real>& SpatialPooler::getBoostedOverlaps() const
   return boostedOverlaps_;
 }
 
+/** helper method that checks SP output is stable for given configuration */
+bool checkUnstableParams_(SpatialPooler &sp) {
+  vector<UInt> input = {1,1,0,0,1,0}; //FIXME auto to size of SP's input
+  if(sp.getInputDimensions().size()>1) return true; //FIXME skip nD as I cannot set auto input correctly
```

Comment:
TODO: I need some "automatic" input. And it must fit to the dimensions specified for the SP's input. 
In most cases it's 1D and hard-coded value is sufficient, but for nD I'd need something as `sp.getDemoInput()`; or is there any getter that has same dimensions as the input?